kfdef/kfctl_openshift_ml-pipelines.yaml

```diff
@@ -14,6 +14,7 @@ spec:
         overlays:
           - metadata-store-mysql
           - ml-pipeline-ui
+          - minio
```

Comment:
Better name would be `object-store-minio`

Reply:
Done, updated name to be `object-store-minio`